### DISCUSSION TITLE
Due to do is actually "done" (my calendar) but still shows as "due to do"

### DISCUSSION THREAD

**Discussion by michael-schramm** [Leantime/leantime#801] [Q&A]
Hi,
I am a bit confused:
Under "my calendar" I can see my due tasks, but I also see some tasks that are already "done" but they still show up as "due".
![image](https://user-images.githubusercontent.com/45664277/180033306-052c7629-7dc6-4876-9a5b-0d3ac315ce61.png)
![image](https://user-images.githubusercontent.com/45664277/180033485-592ab89f-5cf6-4b38-a34f-268529716078.png)

Is this a bug or should it work that way?

**Comment by marcelfolaron** [comment]
Thanks, you are right, this is a bug. The todo should not show up in there
once completed.

**Reply by michael-schramm** [reply]
Should I report it or did you already do it?

**Comment by Anaxagoras-bc** [comment]
I'm demo'ing leantime and this seems to be an issue still?  If a task is marked as done, it is still showing up as "due to-do" in the calendar.

I'd be in favor of tasks that are done having strike through text and/or possibly changing the color of the task card.  Either lighten it, or make it semi transparent.
